roq-deployment/ansible/GCM/tasks/main.yml

```diff
@@ -2,7 +2,7 @@
   file: path=~/.roq state=directory mode=777
 
 - name: Overwrite GCM default config file
-  template: src=GCM.properties.j2 dest='~/.roq/GCM.properties'
+  template: src=GCM.properties.j2 dest='{{roq_temp_path}}/GCM.properties'
```

Comment:
Is it normal that the path "~/.roq" still appears in plenty of places? For example in line 2 of this file.